app/gateway/proxy.py

```diff
@@ -79,33 +80,39 @@ def healthcheck():
     return Response(json.dumps("Up and running"), status=200)
 
 
```

Comment:
Why is it commented out? For later?

Reply:
It's not used at the moment, so I thought we should at least make this obvious.